mockatests/wrappers/wrappers.c

```diff
@@ -142,7 +142,7 @@ int
 wrap_vprintf(const char *format, va_list ap)
 {
 	if (mock_vprintf) {
-		struct printf_data *data = mock();
+		struct printf_data *data = (struct printf_data *)mock();
```

Comment:
Should this also use `mock_ptr_type()`?

Reply:
Yes, probably but I'll catch that in some follow-ups I think